Gruntfile.js

```diff
@@ -0,0 +1,112 @@
+            updateWidgetFiles: {
+                files: [ "./src/**/*" ],
+                tasks: [ "webpack:develop", "compress:dist", "copy:distDeployment", "copy:mpk" ],
+                options: {
+                    debounceDelay: 250,
+                    livereload: true
+                }
+            }
+        },
+
+        compress: {
+            dist: {
+                options: {
+                    archive: "./dist/" + pkg.version + "/" + pkg.widgetName + ".mpk",
+                    mode: "zip"
+                },
+                files: [ {
+                    expand: true,
+                    date: new Date(),
+                    store: false,
+                    cwd: "./dist/tmp/src",
+                    src: [ "**/*" ]
+                } ]
+            }
+        },
+
+        copy: {
+            distDeployment: {
+                files: [ {
+                    dest: "./dist/MxTestProject/deployment/web/widgets",
+                    cwd: "./dist/tmp/src/",
+                    src: [ "**/*" ],
+                    expand: true
+                } ]
+            },
+            mpk: {
+                files: [ {
+                    dest: "./dist/MxTestProject/widgets",
+                    cwd: "./dist/" + pkg.version + "/",
+                    src: [ pkg.widgetName + ".mpk" ],
+                    expand: true
+                } ]
+            }
+        },
+
+        file_append: {
+            addSourceURL: {
+                files: [ {
+                    append: `\n\n//# sourceURL=${pkg.widgetName}.webmodeler.js\n`,
+                    input: `dist/tmp/src/${pkg.widgetName}/widget/${pkg.widgetName}.webmodeler.js`
+                } ]
+            }
+        },
+
+        webpack: {
+            develop: webpackConfig,
+            release: webpackConfigRelease
+        },
+
+        clean: {
+            build: [
+                "./dist/" + pkg.version + "/" + pkg.widgetName + "/*",
+                "./dist/tmp/**/*",
+                "./dist/tsc/**/*",
+                "./dist/testresults/**/*",
+                "./dist/MxTestProject/deployment/web/widgets/" + pkg.widgetName + "/*",
+                "./dist/MxTestProject/widgets/" + pkg.widgetName + ".mpk"
+            ]
+        },
+
+        checkDependencies: {
+            this: {}
+        }
+    });
+
+    grunt.loadNpmTasks("grunt-check-dependencies");
+    grunt.loadNpmTasks("grunt-contrib-clean");
+    grunt.loadNpmTasks("grunt-contrib-compress");
+    grunt.loadNpmTasks("grunt-contrib-copy");
+    grunt.loadNpmTasks("grunt-contrib-watch");
+    grunt.loadNpmTasks("grunt-file-append");
+    grunt.loadNpmTasks("grunt-webpack");
+
+    grunt.registerTask("default", [ "clean build", "watch" ]);
+    grunt.registerTask(
+        "clean build",
+        "Compiles all the assets and copies the files to the dist directory.",
+        [ "checkDependencies", "clean:build", "webpack:develop", /* "file_append", */ "compress:dist", "copy:mpk" ]
```

Comment:
Why comment out file_append

Reply:
Not implemented in beta, still to be discussed with Danny